aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -256,51 +223,6 @@
       "type": "array"
     }
   },
-  "allOf": [
-    {
-      "required": [
-        "Engine"
-      ]
-    },
-    {
-      "anyOf": [
-        {
-          "anyOf": [
-            {
-              "required": [
-                "MasterUserPassword"
```

Comment:
For attributes like this it might be nice to leave them in. The error message returned to the user is fairly straight forward. Indicating the user needs a MasterUserPassword for example.

Another nice benefit is this validation gets done very quickly and on the cfn end. If we remove this validation the customer has to wait for the call to RDS to validate.

Reply:
The motivation is to get rig of messages like `Properties validation failed for resource FooBarBaz with message: #: #: only 1 subschema matches out of 2 #/Iops: failed validation constraint for keyword [minimum]`